pkg/packer/manifests/gcp/packer.json.tmpl

```diff
@@ -13,10 +13,13 @@
     "manifest_output": "manifest.json",
     "network": "",
     "project_id": "",
-    "zone": ""
+    "zone": "",
+    "disk_size": "80",
+    "disk_type": "pd-standard"
   },
   "builders": [
     {
+       "name": "{{(user `distribution`) | lower}}-{{user `distribution_version`}}{{user `build_name_extra`}}",
```

Comment:
is name a valid field here? if so how do I find out these "default" fields I am looking at this btw: `https://www.packer.io/plugins/builders/googlecompute`

Reply:
I agree, its hard to find documentation for `name`. some references: https://www.packer.io/docs/templates/hcl_templates/contextual-variables#build-variables
and
https://www.packer.io/docs/templates/legacy_json_templates/builders#named-builds